### DISCUSSION TITLE
Program to pull all your katas from Codewars into a folder.

### DISCUSSION THREAD

**Discussion by JoseDeFreitas** [codewars/codewars.com#2496] [Show and tell]
EDIT (16/03/2023)

Hello. I created a program that lets you pull all your Codewars completed katas into a folder. The idea behind is that you can have a registry of all your work displayed in a way that is easy to navigate through.

The repository is called [CodewarsLogger](https://github.com/JoseDeFreitas/CodewarsLogger).

It's free and open source. Sharing it in here because I thought someone may use it. I answer most questions in the repository README.md file and in the wiki, but if you want to tell me anything else, you can use this discussion.

**Comment by hobovsky** [comment]
Please refer to the last rule of Codewars Code of Conduct: https://docs.codewars.com/community/rules#rules as it could be relevant for your script.

I think that repository of the script should make it clear that public repositories of solutions for Codewars kata are generally discouraged and frowned upon by community, as they very often serve as a source for dishonest users who copy solutions and submit as their own. It's highly recommended to make such repositories private and include a note in its README that copying solutions and submitting them to get points can get a user into troubles, ranging from invalidation of a solution to a ban when heavily abused.

**Reply by JoseDeFreitas** [reply]
Hello. I've set my solutions' repository to private. Moreover, I've added a clear notice at the top of the README.md file mentioning the Codewars Code of Conduct. Also, all references to public repositories have been removed or replaced by the discouraged use of them (both in the README.md file and in the wiki pages). Thank you for your response.

**Comment by petermoyano** [comment]
Awesome idea, but I couldn't make it work. When I run the "dotnet run" command in the terminal, I get this error:
**Unhandled exception. OpenQA.Selenium.DriverServiceNotFoundException: The file ./geckodriver does not exist.**
... Program.cs: line 50
Don't really know how to solve this issue. Would love to hear if you have any ideas!
If not, thanks anyway!

**Reply by JoseDeFreitas** [reply]
Hello. Thank you for your feedback. I've opened an issue from your comment in my repository (issue [#1](https://github.com/JoseDeFreitas/CodewarsGitHubLogger/issues/1)) to treat everything in there, in order to keep this discussion strictly related to the inclusion and acceptance of the project.
